rust/capture/src/config.rs

```diff
@@ -110,4 +110,6 @@ pub struct KafkaConfig {
     pub kafka_producer_max_retries: u32,
     #[envconfig(default = "all")]
     pub kafka_producer_acks: String,
+    #[envconfig(default = "60000")]
```

Comment:
```suggestion
    #[envconfig(default = "20000")]
```

Let's go straight away for a lower default